utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -223,5 +223,25 @@ class HtmlParser private constructor(
         policyOppiaTagActionListener
       )
     }
+
+    /**
+     * Returns a new [HtmlParser] with an optionally specified [CustomOppiaTagActionListener] and
+     * [PolicyOppiaTagActionListener] for handling custom Oppia tag events.
```

Comment:
```suggestion
     * [PolicyOppiaTagActionListener] for handling custom Oppia tag events. Note that Oppia image loading is specifically not supported (see the other [create] method if image support is needed).
```

(Will need wrapping)